Cargo.toml

```diff
@@ -16,13 +16,12 @@ category = ["cryptography", "data-structures"]
 publish = true
 
 [features]
-default = ["ics23", "std", "metrics"]
```

Comment:
Why don't we want this to be a default feature any more?

Reply:
thanks for catching this, unstaged changes slipped through!